docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+# Organizations
+
+## Problems
+ 1.  Cluster administrators want to allocate a pool of resources to a subset of their company, and delegate the authority to manage how that pool is assigned to an organization administrator.
+ 2.  An organization administrator needs to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  Cluster administrators want to delegate control over groups of users to an organization administrator - that organization administrator should be able to manage the access of a set of users to individual projects under that organization umbrella.
+ 4.  Cluster administrators want to allow self-service of users on the cluster, but the total allocated resources those self service users can get access to is limited (to prevent abuse / unfair use of resources).
+
+We can do this by introducing the concept of project ownerhsip.  A Project is owned by at most one (Organization xor User).  We'll also introduce the idea of a `ClusterResourceQuota` object that can be associated with an organization or a user. An Organization can have multiple org-owners who are allowed to manage quota allocation to owned projects and OrgGroups (groups that scoped to projects owned by an Organization).
+
+### Open Questions
+ 3.  Can we punt on adding users as organization members?  Still don't see a clear answer on this.
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
+ 4.  Can we punt on project transfering?  Sounds like yes.  A cluster-admin could transfer ownership.
+
+     An org-owner shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
+
+ 6.  What things can org-owners do?
+
+ 	There's a list below proposing things.
+
+
+## ClusterResourceQuota
+`ClusterResourceQuota` mirrors `ResourceQuota`, but at the cluster-scope.  (We can't call it that because `ResourceQuota` is a namespace-scope resource and it can't be in both scope.)  A `ClusterResourceQuota` has an optional owner that is either a user or an organization and an optional label selector.  An empty label selector is `true`.  If both are set, then the `ClusterResourceQuota` applies to all namespaces matching the selector **and** owned by the owner.
+
+The existing quota admission controller would remain in place, but no limits would be set by default.  If a project owner decided that a particular project was starving the rest, he could assign hard limits to that Project's `ResourceQuota` to prevent it from taking more.  An additional admission controller would be responsible for doing the rollup across all projects in the Organization and preventing over-use of resources.
```

Comment:
I do not understand what you mean by limits.

Are you referring to a `ResourceQuota.Spec.Hard["cpu"]` would be left unset?  Right now, quota only counts resource usage for resources that are enumerated.  For example, if quota only tracks cpu, it does not count usage of memory.

I think I read this that you have an admission controller that intercepts changes to `ResourceQuota.Spec.Hard` and ensures that it is not given out more quota than what `ClusterResourceQuota.Spec.Hard` allows.  That makes sense if I read that correctly.